### PR TITLE
doc:add config for hot region history and add command for pd ctl 

### PR DESCRIPTION
**Pull request by qidi1**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

add config for hot region history and add command for pd ctl.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v5.4 (TiDB 5.3 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- Chinese Version: pingcap/docs-cn/pull/7163
- Other reference link(s):
    Relative docs:
    * Chinese vergion: pingcap/docs-cn/pull/7164
    * English vergion: pingcap/docs/pull/6624
    
    Feature tracking Issue: 
    * pingcap/tidb/pull/25281
   
### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: pd-control.md [hunk 1]
```diff
@@ -376,6 +376,63 @@ Usage:
...
+    HotRegionType ... 
+    HotDegree     ... 
+    FlowBytes     ... 
+    KeyRate       ... 
+    QueryRate     ... 
+    StartKey      ... 
+    EndKey        ... 
+    EncryptionMeta ...
+  }
+]
```
Comment (hfxsd) [inline pd-control.md:435]:
```suggestion
```

File: pd-control.md [hunk 2]
```diff
@@ -376,6 +376,63 @@ Usage:
...
+    IsLearner     ... 
+    HotRegionType ... 
+    HotDegree     ... 
+    FlowBytes     ... 
+    KeyRate       ... 
+    QueryRate     ... 
+    StartKey      ... 
+    EndKey        ... 
+    EncryptionMeta ...
+  }
```
Comment (hfxsd) [inline pd-control.md:396]: 这行下面，中文版有 ......  请确认是否要加上？ @qidi1

File: pd-control.md [hunk 3]
```diff
@@ -376,6 +376,63 @@ Usage:
 >> hot read                             // Display hot spot for the read operation
 >> hot write                            // Display hot spot for the write operation
 >> hot store                            // Display hot spot for all the read and write operations
+>> hot history 1634478065 1634650865    // Display history hot spot
```
[the remark this answers is not in the document]
Comment (qidi1) [reply pd-control.md]: 已经更新好了，使用了具体的返回示例作为例子，并对使用注释进行了修改，麻烦帮忙review一下下~

File: pd-control.md (hunk 2 above)
Comment (qidi1) [reply pd-control.md]: 不应该出现，同上。

**Review by hfxsd — APPROVED** [review]
LGTM

**Review by nolouch — APPROVED** [review]

**Comment by nolouch** [comment]
ptal @en-jin19

File: pd-configuration-file.md [hunk 4]
```diff
@@ -283,6 +283,20 @@ Configuration items related to scheduling
...
+### `hot-regions-write-interval`
+
++ The time interval for PD to store hot Region information.
++ Default value: `10m`
+
+> **Note:**
+> 
+>Information about the hot Region is updated every three minutes. If the interval is set to less than three minutes, updates during the interval might be meaningless.
+
+### `hot-regions-reserved-days`
```
Comment (TomShawn) [inline pd-configuration-file.md:295]:
```suggestion
### `hot-regions-reserved-days` <span class="version-mark">New in v5.4.0</span>
```

File: pd-configuration-file.md [hunk 5]
```diff
@@ -283,6 +283,20 @@ Configuration items related to scheduling
...
+
++ How many days the hot Region information is retained.
```
Comment (TomShawn) [inline pd-configuration-file.md:297]:
```suggestion
+ Specifies how many days the hot Region information is retained.
```

File: pd-control.md [hunk 6]
```diff
@@ -366,16 +366,58 @@ Usage:
...
+      "flow_bytes": 0,
+      "key_rate": 0,
+      "query_rate": 305,
+      "start_key": "7480000000000000FF5300000000000000F8",
+      "end_key": "7480000000000000FF5600000000000000F8"
+    },
+    ...
+  ]
+}
+>> hot history 1629294000000 1631980800000 hot_region_type read region_id 1,2,3 store_id 1,2,3 peer_id 1,2,3 is_leader true is_learner true // Display history hot spot for the specified period with more conditions
```
Comment (TomShawn) [inline pd-control.md:400]:
```suggestion
>> hot history 1629294000000 1631980800000 hot_region_type read region_id 1,2,3 store_id 1,2,3 peer_id 1,2,3 is_leader true is_learner true // Display history hotspot for the specified period with more conditions
```

File: pd-control.md [hunk 7]
```diff
@@ -366,16 +366,58 @@ Usage:
...
 Usage:
 
 ```bash
->> hot read                             // Display hot spot for the read operation
->> hot write                            // Display hot spot for the write operation
->> hot store                            // Display hot spot for all the read and write operations
+>> hot read                                // Display hot spot for the read operation
+>> hot write                               // Display hot spot for the write operation
+>> hot store                               // Display hot spot for all the read and write operations
+>> hot history 1629294000000 1631980800000 // Display history hot spot for the specified period (milliseconds)
```
Comment (TomShawn) [inline pd-control.md:379]:
```suggestion
>> hot history 1629294000000 1631980800000 // Display history hot spot for the specified period (milliseconds). 1629294000000 is the start time and 1631980800000 is the end time.
```

**Review by TomShawn — APPROVED** [review]

**Pull request merged**